helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -674,6 +681,9 @@ otelCollector:
   # existing fields can be disabled by setting them to `null`.
   config: {}
 
+# Specifies whether to apply for k8s cluster with windows worker node.
+isWindows: false
+
```

Comment:
Let's move this part under `Extra system configuration` section

Reply:
Moved `isWindows` flag.